installers/install.sh

```diff
@@ -342,17 +342,11 @@ esac
 
 # Write install file
 STATEPATH=$INSTALLDIR/$STATEEXE
-CONFIGDIR=$($STATEPATH "export" "config" "--filter=dir")
+CONFIGDIR=$($STATEPATH "--output=simple" "export" "config" "--filter=dir")
 echo "install.sh" > $CONFIGDIR/"installsource.txt"
 
-# Check if the installation is in $PATH, if so we also check if the activate
-# flag was passed and attempt to activate the project
-if [ "`dirname \`which $STATEEXE\` 2>/dev/null`" = "$INSTALLDIR" ]; then
-  info "State Tool installation complete."
-fi
-
-
-info "State Tool installation complete."
+info "State Tool successfully installed."
+info "Reminder: Start a new shell in order to start using the State Tool."
```

Comment:
Can we highlight both of these like we do on Windows (we're doing this elsewhere in the sh)

Reply:
The `info` function does make it bold already. Would you like a different style like `warn` or a yellow coloring like the Windows script?